storage/Utils/Logger.cc

```diff
@@ -58,6 +61,41 @@ namespace storage
     }
 
 
+    class LogfileLogger : public Logger
+    {
+    public:
+
+	virtual void write(LogLevel log_level, const std::string& component, const std::string& file,
+			   int line, const std::string& function, const std::string& content) override;
+
+    };
+
+
+    void
+    LogfileLogger::write(LogLevel log_level, const std::string& component, const std::string& file,
+			 int line, const std::string& function, const std::string& content)
+    {
+	FILE* f = fopen("/var/log/libstorage.log", "ae");
```

Comment:
Since this is only used once (in stark contrast to previously where it was in each of the examples), I wouldn't move it to a #define in this case; this would just scatter the information around (once for the #define, once for the fopen() call where that #define'd symbol would be used).